app/kumactl/cmd/get/get_meshes.go

```diff
@@ -56,7 +57,8 @@ func printMeshes(meshes *mesh.MeshResourceList, out io.Writer) error {
 				mesh := meshes.Items[i]
 
 				return []string{
-					mesh.GetMeta().GetName(), // NAME
+					mesh.GetMeta().GetName(),            // NAME
+					table.OnOff(mesh.Spec.Mtls.Enabled), // MTLS
```

Comment:
since `Mtls` is optional (`*Mesh_Mtls`), it would be more safe to call `mesh.Spec.Mtls.GetEnabled()`